MeetupPlatformApi/Entities/UserEntity.cs

```diff
@@ -13,4 +13,6 @@ public class UserEntity
 
     [Required]
```

Comment:
Looks like in our other PR (the one about IEntityTypeConfiguration) we completely forgot to remove all entity annotations. Can You please do it in the scope of this PR?